dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -252,6 +252,114 @@ html, body {
   }
 }
 
+/* Styling that keeps the small footer consistent with other page elements */
+$codeApp-left-margin: 25px;
+$small-footer-horizontal-margin: 15px;
+$small-footer-standard-width: 400px;
+
+/* Shared styles so flyout looks like footer */
+.small-footer, #copyright-flyout, #more-menu {
+  background-color: #ddd; /* Matches blockly flyout */
+  color: $light_gray;
+  font-size: 14px;
+  a {
+    &:link {
+      font-family: $gotham-bold;
+      font-weight: normal;
+      color: $light_gray;
+      text-decoration: none;
+    }
+    &:visited {
+      font-family: $gotham-bold;
+      font-weight: normal;
+      color: $light_gray;
+      text-decoration: none;
+    }
+    &:hover {
+      font-family: $gotham-bold;
+      font-weight: normal;
+      color:$dark_charcoal;
+      background-color: transparent;
+      text-decoration: underline;
+    }
+    &:active {
+      font-family: $gotham-bold;
+      font-weight: normal;
+      color:$light_gray;
+      background-color: transparent;
+      text-decoration: none;
+    }
+  }
+}
+
+.small-footer {
+  position: absolute;
+  z-index: 3;
+  bottom: 0;
+  width: $small-footer-standard-width;
+  overflow: hidden;
+  padding-left: $codeApp-left-margin;
+  padding-top: 3px;
+  padding-bottom: 3px;
+  margin-top: 5px; /* Used in javascript to position #belowVisualization */
+
+  form {
+    float: left;
+    margin-right: $small-footer-horizontal-margin;
+  }
+
+  select {
+    width: 80px;
+    font-size:11px;
+    height: 18px;
+    margin-bottom: 0;
+  }
+
+  small {
+    display: block;
+    float: right;
+    margin-right: $small-footer-horizontal-margin;
+    padding-top: 3px;
+    white-space: nowrap;
+  }
+}
+
+#copyright-flyout {
+  display: none; /* Invisible until summoned. */
+  position: absolute;
+  z-index: 2;
+  bottom: 0;
+  left: 0;
+  width: 625px;
+  max-width: 50%;
+  min-width: 300px;
+  padding: 1em;
+}
+
+#more-menu {
+  display: none;
+  position: absolute;
+  z-index: 1;
```

Comment:
I rather like treating z-indexes like colors; giving them descriptive rather than referential names combined with offsets.

In the interest of focus, I'll reopen the definitions, but I'd like to make a note somewhere to go through the z-indexes everywhere and standardize them a bit. What's the best protocol for that?